src/components/HomePage.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] Kindly enable **all** disabled linter checks in your code to enhance a cleaner and robust code, as requested in the previous review.